converter/src/main/java/gov/cms/qpp/conversion/encode/MeasureDataEncoder.java

```diff
@@ -53,14 +47,19 @@ protected void internalEncode(JsonWrapper wrapper, Node node) {
 	 *
 	 * @return intialized measure type map
 	 */
-	private Map<String, String> initializeMeasureTypeMap() {
-		Map<String, String> measureTypeMapper = new HashMap<>();
+	private Map<SubPopulationLabel, String> initializeMeasureTypeMap() {
+		Map<SubPopulationLabel, String> measureTypeMapper = new HashMap<>();
 		final String eligiblePopulation = "eligiblePopulation";
 
-		measureTypeMapper.put(SubPopulations.NUMER, "performanceMet");
-		measureTypeMapper.put(SubPopulations.DENOM, eligiblePopulation);
-		measureTypeMapper.put(SubPopulations.DENEX, "eligiblePopulationExclusion");
-		measureTypeMapper.put(SubPopulations.DENEXCEP, "eligiblePopulationException");
+		measureTypeMapper.put(SubPopulationLabel.NUMER, "performanceMet");
+		measureTypeMapper.put(SubPopulationLabel.DENOM, eligiblePopulation);
+		measureTypeMapper.put(SubPopulationLabel.DENEX, "eligiblePopulationExclusion");
+		measureTypeMapper.put(SubPopulationLabel.DENEXCEP, "eligiblePopulationException");
+
+//		measureTypeMapper.put(SubPopulationLabel.NUMER.name(), "performanceMet");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)